.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
       - id: go-version
-        run: echo "version=$(cat .go-version)" >> $GITHUB_OUTPUT
+        run: echo "version=$(cat .go-version)" >> "$GITHUB_OUTPUT"
```

Comment:
What influenced this change?

Reply:
shellcheck via actionlint. See https://www.shellcheck.net/wiki/SC2086

Reply:
failure is here https://github.com/hashicorp/terraform-provider-vault/actions/runs/5204927717/jobs/9389779657